spec/jobs/organization_event_job_spec.rb

```diff
@@ -42,9 +42,23 @@
 
         OrganizationEventJob.perform_now(payload)
 
-        expect { @organization.users.find(uid: @user.uid) }.to raise_error(ActiveRecord::RecordNotFound)
+        expect(@organization.users).not_to include(@user)
         expect(assignment.reload.creator_id).not_to eq(@user.id)
       end
+
+      it "deletes user from multiple organizations (classrooms) mapped to the same GitHub organization" do
```

Comment:
Oof this is another one of those times when I wish we didn't have 2 different concepts of "organization" in the backend :C

Reply:
Yes! I can't decide if it's going to get better or worse if we move into the monolith.